test/python/test_bdd_nexus.py

```diff
@@ -0,0 +1,494 @@
+import pytest
+from pytest_bdd import given, scenario, then, when, parsers
+
+from collections import namedtuple
+import os
```

Comment:
Does not seem to be used? Python has very rich tooling in terms of linters, checkers etc... maybe install a few?

Reply:
Will look into adding extra linter(s) to the commit hook, but have removed the unused import for now.